lib/fluent/plugin/buf_file.rb

```diff
@@ -67,33 +73,42 @@ def configure(conf)
 
         @@buffer_paths[@path] = type_of_owner
 
-        if File.exist?(@path)
-          if File.directory?(@path)
+        if File.exist?(@path) && File.directory?(@path) || !File.exist?(@path) && !@path.include?('.*') # directory
```

Comment:
Yes, because this clause must NOT match the condition of `File.exist?(@path) && !File.directory?(@path)`.